src/main/resources/config/application.yaml

```diff
@@ -3,9 +3,9 @@ spring:
     name: directory-server
 
   cloud:
+    function:
+      definition: consumeStudyUpdate;consumeElementUpdate
     stream:
-      function:
-        definition: consumeStudyUpdate;consumeElementUpdate
```

Comment:
OK saw https://itsmariodias.medium.com/to-anyone-referring-this-article-please-note-that-since-spring-cloud-stream-v3-1-3-7d6961f961a3

> To anyone referring this article, please note that since spring-cloud-stream v3.1.3 the property spring.cloud.stream.function.definition has been deprecated and moved to spring.cloud.function.definition instead."